src/ports/postgres/modules/utilities/path.py_in

```diff
@@ -40,7 +40,7 @@ m4_changequote(`<!', `!>')
 
 def path(schema_madlib, source_table, output_table, partition_expr,
          order_expr, pattern_expr, symbol_expr, agg_func,
-         persist_rows=None, **kwargs):
+         persist_rows=None, overlapping_patterns=None, **kwargs):
```

Comment:
I don't see why setting the default value to False is better. As you said, the user can still explicitly set to NULL and we still need to account for that case.